container_images/gointegtest/main.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+
+echo "Running daisy integration test workflow"
+cd /  # Prow will put us in the dir with repo checked out.
+
+GCS_PATH="${GCS_BUCKET}/`date '+%s'`"
+
+if [[ "$JOB_TYPE" == "presubmit" ]]; then
+  GIT_REF="pull/${PULL_NUMBER}/head"
+else
+  GIT_REF="$PULL_BASE_REF"
+fi
+
+/daisy -project "$PROJECT" -zone "$ZONE" -var:repo_name="$REPO_NAME" \
+  -var:repo_owner="$REPO_OWNER" -var:git_ref="$GIT_REF" \
+  -var:gcs_path="$GCS_PATH" integ-test-all.wf.json
+RET=$?
+
+gcloud auth activate-service-account --key-file="$GOOGLE_APPLICATION_CREDENTIALS"
+gsutil cp "$GCS_PATH"/go-test*txt ./
```

Comment:
Consider archiving go-test-$platform.txt. the junit converter is lossy, eg: https://github.com/jstemmer/go-junit-report/issues/108

An easy solution, I think, is to do something like `-var:gcs_path="$ARTIFACTS/daisy" `, since I think this would keep serial console output as well.

Reply:
serves me for giving a concrete solution to a general suggestion :)